git-machete.rb

```diff
@@ -4,7 +4,8 @@ class GitMachete < Formula
   homepage "https://github.com/VirtusLab/git-machete"
   url "https://pypi.org/packages/source/g/git-machete/git-machete-3.12.0.tar.gz"
   sha256 "b999ff0ead4b856436ee74aae394ae3c68941fab8286256488304382d0fc8452"
-  depends_on "python"
+  license "MIT"
+  depends_on "python@3.10"
```

Comment:
Hmmm would it be possible to stick to just `python`? 🤔 I'd just want to prevent Python hell as much as possible on users' machines (so if they have another version of Python installed, let's stick to that)

Reply:
@PawelLipski Unfortunately it's not possible, you have to provide the version or otherwise the formula wont pass the audit check `brew audit --new <formula>`

Reply:
☝🏻 meh, probably this is wrong, see https://github.com/VirtusLab/git-machete/issues/665#issuecomment-1262239316

Reply:
@PawelLipski I mean you are right, it doesn't have to be maintained since it is already introduced to the homebrew-core but for the sake of documentation and maybe some feature issues it can be up-to-date, wdyt?

Reply:
I'd say, for the sake of people **who already installed from tap**... I doubt they'll automatically get the updates from core git-machete; they're probably bound to `VirtusLab/git-machete` tap for good (unless they do some magic reconfig/reinstall, which we won't suggest not to complicate stuff/lose users)